frontend/src/API/use/useInterpreter.ts

```diff
@@ -14,15 +16,31 @@ export const ANNOTATE_LIMIT = 1024;
 export function useInterpreter(originalContent: string, originalTitle: string | undefined, id: number, type: 'post' | 'comment') {
     const api = useAPI();
     const contentRef = useRef<HTMLDivElement>(null);
-    const [currentMode, setCurrentMode] = React.useState<AltContentType | undefined >();
+    const [currentMode, setCurrentMode] = React.useState<AltContentType | undefined>();
     const [cachedTitleTranslation, setCachedTitleTranslation] = useState<string | undefined>();
-    const [cachedContentTranslation, setCachedContentTranslation] = useState< string | undefined>();
+    const [cachedContentTranslation, setCachedContentTranslation] = useState<string | undefined>();
     const [streamingAnnotation, setStreamingAnnotation] = useState<string | undefined | null>(); // null is a special value indication that content is being fetched
     const [cachedAnnotation, setCachedAnnotation] = useState<string | undefined>();
     const [streamingAltTranslation, setStreamingAltTranslation] = useState<string | undefined | null>();  // null is a special value indication that content is being fetched
     const [cachedAltTranslation, setCachedAltTranslation] = useState<string | undefined>();
     const [inProgress, setInProgress] = useState<boolean>(false);
 
+    const calcStrippedOriginalContentLength = useLazy(() =>
+        stripHtml(originalContent,
+            {
+                stripTogetherWithTheirContents: [
+                    'script', // default
```

Comment:
Also `img` and `video` to stop showing annotate for image-only posts

Reply:
`img` and `video` don't have content.